ios/Classes/AblyFlutter.m

```diff
@@ -814,16 +792,13 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 }
 
 #pragma mark - Push Notifications Registration - UIApplicationDelegate
-/// Save the deviceToken provided so we can pass it to the first Ably client which gets created, in createRealtime or createRest.
+
 -(void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
-    // Set deviceToken on all existing Ably clients, and a property which used for all future Ably clients.
```

Comment:
OOI, why have you decided to remove these comments?

Reply:
Because the code setting deviceToken on all existing Ably clients was removed. I've left the comment about the single client though.